android/sdk/src/main/java/org/jitsi/meet/sdk/PictureInPictureModule.java

```diff
@@ -27,11 +27,15 @@
 import com.facebook.react.bridge.ReactApplicationContext;
 import com.facebook.react.bridge.ReactContextBaseJavaModule;
 import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.module.annotations.ReactModule;
 
+@ReactModule(name = PictureInPictureModule.NAME)
 class PictureInPictureModule
     extends ReactContextBaseJavaModule {
 
-    private final static String TAG = "PictureInPicture";
+    public static final String NAME = "PictureInPicture";
+
+    private static final String TAG = NAME;
```

Comment:
What is the purpose of TAG in general?

Reply:
It's a convention used for logging. You can then filter logs by tag with adb.